### PR TITLE
Add business process in merge process

### PR DESCRIPTION
**Pull request by FranckLecuyer**
Branch 'add_business_process_in_merge_process' is built on branch 'add-configs'

### CODE REVIEW

File: src/main/java/org/gridsuite/merge/orchestrator/server/repositories/IgmEntityKey.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (ChamseddineBhd) [inline src/main/java/org/gridsuite/merge/orchestrator/server/repositories/IgmEntityKey.java:2]: I think we should keep 2020 as the file was created in 2020

File: src/main/java/org/gridsuite/merge/orchestrator/server/dto/ProcessConfig.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (ChamseddineBhd) [inline src/main/java/org/gridsuite/merge/orchestrator/server/dto/ProcessConfig.java:2]: I think we should keep 2020 as the file was created in 2020

File: src/main/java/org/gridsuite/merge/orchestrator/server/dto/MergeConfig.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (ChamseddineBhd) [inline src/main/java/org/gridsuite/merge/orchestrator/server/dto/MergeConfig.java:2]: same remark

File: src/main/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorService.java [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (ChamseddineBhd) [inline src/main/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorService.java:2]: same remark

File: src/main/java/org/gridsuite/merge/orchestrator/server/MergeEventService.java [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (ChamseddineBhd) [inline src/main/java/org/gridsuite/merge/orchestrator/server/MergeEventService.java:2]: I think we should keep 2020 as the file was created in 2020

File: src/test/java/org/gridsuite/merge/orchestrator/server/ProcessConfigControllerTest.java [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (ChamseddineBhd) [inline src/test/java/org/gridsuite/merge/orchestrator/server/ProcessConfigControllerTest.java:2]: same remark

File: src/test/java/org/gridsuite/merge/orchestrator/server/MergeTest.java [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (ChamseddineBhd) [inline src/test/java/org/gridsuite/merge/orchestrator/server/MergeTest.java:2]: same remark

File: src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
```
Comment (ChamseddineBhd) [inline src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java:2]: Same remark

File: src/main/java/org/gridsuite/merge/orchestrator/server/dto/MergeConfig.java [hunk 9]
```diff
@@ -24,5 +25,7 @@
```
Comment (ChamseddineBhd) [inline src/main/java/org/gridsuite/merge/orchestrator/server/dto/MergeConfig.java:25]: I think we don't use this class anymore : to be deleted

**Review by ChamseddineBhd — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/merge/orchestrator/server/repositories/IgmEntityKey.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/merge/orchestrator/server/repositories/IgmEntityKey.java]: Done

File: src/main/java/org/gridsuite/merge/orchestrator/server/dto/ProcessConfig.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/merge/orchestrator/server/dto/ProcessConfig.java]: Done

File: src/main/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorService.java (hunk 4 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorService.java]: Done

File: src/main/java/org/gridsuite/merge/orchestrator/server/MergeEventService.java (hunk 5 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/merge/orchestrator/server/MergeEventService.java]: Done

File: src/test/java/org/gridsuite/merge/orchestrator/server/ProcessConfigControllerTest.java (hunk 6 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/merge/orchestrator/server/ProcessConfigControllerTest.java]: Done

File: src/test/java/org/gridsuite/merge/orchestrator/server/MergeTest.java (hunk 7 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/merge/orchestrator/server/MergeTest.java]: Done

File: src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java (hunk 8 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorIT.java]: Done

File: src/main/java/org/gridsuite/merge/orchestrator/server/dto/MergeConfig.java (hunk 9 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/merge/orchestrator/server/dto/MergeConfig.java]: class deleted

**Review by ChamseddineBhd — APPROVED** [review]

**Pull request merged**